tests/test_aggregate.py

```diff
@@ -138,6 +138,10 @@ def test_calendars(self):
             as_dataset=True,
         )
 
+        assert isinstance(
+            xs.climatological_mean(convert_calendar(ds, "standard")),
+            cftime.Datetime360Day,
+        )
```

Comment:
what ? Standard -> Datetime360Day ?

My commit changed the logic here anyway.

Reply:
I think that's some bad copy-paste 😄